setup.sh

```diff
@@ -2,19 +2,26 @@
 
 POOL6="64:ff9b::/96"
 LOWEST_IPV6_MTU=1280
+HANDLE_RST_DURING_FIN_RCV="false"
 
 while [ $# -gt 0 ]; do
   case "$1" in
     --pool6=*)
       POOL6="${1#*=}"
       ;;
     --lowest-ipv6-mtu=*)
-      LOWEST_IPV6_MTU="${1#*=}"
+      LOWEST_IPV6_MTU="true"
+      ;;
+    --handle-rst-during-fin-rcv=*)
```

Comment:
Probably shouldn't contain `=`
```suggestion
    --handle-rst-during-fin-rcv*)
```